mypy/types.py

```diff
@@ -1693,6 +1743,9 @@ def item_str(name: str, typ: str) -> str:
                 suffix = ', fallback={}'.format(t.fallback.accept(self))
         return 'TypedDict({}{}{})'.format(prefix, s, suffix)
 
+    def visit_literal_type(self, t: LiteralType) -> str:
+        return 'Literal[{}]'.format(t.value)
```

Comment:
Maybe use `repr()` here so that string literals that contain unusual characters won't cause trouble. It would also match the source syntax.